src/site/assets/robots.txt

```diff
@@ -93,8 +93,8 @@ Disallow: /
 User-Agent: *
 Disallow: /analytics-opt-out.html
 Disallow: /cgi-bin/
-Disallow: /scorecard
 Disallow: /drupal
+Disallow: /performance-dashboard
```

Comment:
In case we need to revert this too, let's just be ready to open a PR to revert the revert of the revert of the revert

Reply:
hahahaha